code-challenges.test.js

```diff
@@ -35,12 +43,26 @@ const num2 = 0
 const num3 = -7
 // Expected output: "-7 is not divisible by three"
 
-
-// b) Create the function that makes the test pass.
-
-
-
-// --------------------2) Create a function that takes in an array of words and returns an array with all the words capitalized.
+//  describe("divisibleBy", () => {
+//        it("takes a number as an argument and decides if the number is evenly divisble by three or not", () => {
+//            expect(divisibleBy("15").toBe("is divisible by three"))
+//            expect(divisibleBy("0").toBe("is divisible by three"))
+//            expect(divisibleBy("-7").toBe("is not divisible by three"))
+//        })
```

Comment:
Make sure to pay attention to the data type you are passing into the function.  Also, I would usually recommend using the .toEqual for most of the tests we will run.  Eventually we will give you a different matcher to try, but provide documentation and it has a specific use case.  Otherwise, .toEqual is checking strict equality which is usually our goal with a test.